packages/jsii-kernel/lib/serialization.ts

```diff
@@ -168,10 +168,33 @@ export const SERIALIZERS: {[k: string]: Serializer} = {
       // Just whatever. Dates will automatically serialize themselves to strings.
       return value;
     },
-    deserialize(value, optionalValue) {
+    deserialize(value, optionalValue, host) {
       // /!\ Top-level "null" will turn to undefined, but any null nested in the value is valid JSON, so it'll stay!
       if (nullAndOk(value, optionalValue)) { return undefined; }
-      return value;
+      if (isWireMap(value)) {
```

Comment:
can you add a comment explaining why would a value be a "wire map" and what it is?